tools/modules/initialize/check_distro_status.sh

```
@@ -41,3 +41,34 @@ function check_distro_status() {
+#
+module_options+=(
+	["info_wait_autocontinue,author"]="@igorpecovnik"
+	["info_wait_autocontinue,ref_link"]=""
+	["info_wait_autocontinue,feature"]="info_wait_autocontinue"
+	["info_wait_autocontinue,desc"]="Display a warning with a gauge for 10 seconds then continue"
+	["info_wait_autocontinue,group"]="Initialize"
+)
+#
+function info_wait_continue() {
+	local message="$1"
+	local next_action="$2"
```

Shellcheck (0.10.0)

[warning] next_action appears unused. Verify use (or export if used externally).

(SC2034)